src/clinical/donor-repo.ts

```diff
@@ -398,6 +404,16 @@ const ExposureSchema = new mongoose.Schema(
 
 ExposureSchema.index({ exposureId: 1 }, { unique: true, spare: true });
 
+const ComorbiditySchema = new mongoose.Schema(
+  {
+    comorbidityId: { type: Number },
+    clinicalInfo: {},
+  },
+  { _id: false },
+);
+
+ComorbiditySchema.index({ comorbidityId: 1 }, { unique: true, spare: true });
```

Comment:
it's `sparse: true` not `spare` I see this typo in a couple lines above too

Reply:
👍

Reply:
I think we also need to check production to see if any of these indexes were created incorrectly, they need to be deleted and recreated or errors will occur when people submit data

Reply:
Note: Verified in dev and qa, because of the typo, `family_history` and `exposure` indexes were not created in mongo, prod has typo in `family_history` only, so prod's fh index was most likely not created either. The results are in dev and qa, fh and exposure will not have the `unique` and `sparse`  schema constraints, data can still be uploaded, the missing mongo indexes should be created when this PR is merged to dev and qa, same for prod.